core/os/device/gpu_counter_descriptor.proto

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package device;
+option java_package = "com.google.gapid.proto.device";
+option java_outer_classname = "GpuProfiling";
+option go_package = "github.com/google/gapid/core/os/device";
+// cc_package
+// js_path = "proto";
+
+// This file is a mirror of the gpu_counter_descriptor.proto from
```

Comment:
If it is a mirror, can you just deserialize it into the new proto rather than the old and then translate? If they are the same than they should be binary compatible.